rafx-api/src/backends/gles2/internal/gl_context/native.rs

```diff
@@ -0,0 +1,1175 @@
+            self.gles2.ClearColor(r, g, b, a);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_clear_depthf(
+        &self,
+        d: f32,
+    ) -> RafxResult<()> {
+        unsafe {
+            self.gles2.ClearDepthf(d);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_clear_stencil(
+        &self,
+        s: i32,
+    ) -> RafxResult<()> {
+        unsafe {
+            self.gles2.ClearStencil(s);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_clear(
+        &self,
+        mask: u32,
+    ) -> RafxResult<()> {
+        unsafe {
+            self.gles2.Clear(mask);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_finish(&self) -> RafxResult<()> {
+        unsafe {
+            self.gles2.Finish();
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_create_framebuffer(&self) -> RafxResult<FramebufferId> {
+        unsafe {
+            let mut framebuffer = 0;
+            self.gles2.GenFramebuffers(1, &mut framebuffer);
+            self.check_for_error()?;
+            Ok(FramebufferId(framebuffer))
+        }
+    }
+
+    pub fn gl_destroy_framebuffer(
+        &self,
+        framebuffer_id: FramebufferId,
+    ) -> RafxResult<()> {
+        unsafe {
+            self.gles2.DeleteFramebuffers(1, &framebuffer_id.0);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_create_texture(&self) -> RafxResult<TextureId> {
+        unsafe {
+            let mut texture = 0;
+            self.gles2.GenTextures(1, &mut texture);
+            self.check_for_error()?;
+            Ok(TextureId(texture))
+        }
+    }
+
+    pub fn gl_destroy_texture(
+        &self,
+        texture_id: TextureId,
+    ) -> RafxResult<()> {
+        unsafe {
+            self.gles2.DeleteTextures(1, &texture_id.0);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_create_buffer(&self) -> RafxResult<BufferId> {
+        unsafe {
+            let mut buffer = 0;
+            self.gles2.GenBuffers(1, &mut buffer);
+            self.check_for_error()?;
+            Ok(BufferId(buffer))
+        }
+    }
+
+    pub fn gl_destroy_buffer(
+        &self,
+        buffer_id: BufferId,
+    ) -> RafxResult<()> {
+        unsafe {
+            self.gles2.DeleteBuffers(1, &buffer_id.0);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_bind_buffer(
+        &self,
+        target: GLenum,
+        buffer_id: BufferId,
+    ) -> RafxResult<()> {
+        unsafe {
+            self.gles2.BindBuffer(target, buffer_id.0);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_bind_framebuffer(
+        &self,
+        target: GLenum,
+        framebuffer_id: FramebufferId,
+    ) -> RafxResult<()> {
+        unsafe {
+            self.gles2.BindFramebuffer(target, framebuffer_id.0);
+            self.check_for_error()
+        }
+    }
+
+    pub fn gl_vertex_attrib_pointer(
```

Comment:
Missing a function for `glVertexAttribDivisor`.

Reply:
Doesn't exist in ES2